lab5/utils/ConsoleManager.java

```diff
@@ -6,17 +6,34 @@ public class ConsoleManager {
     private final CommandManager commandManager;
     private final Scanner scanner;
 
-    public ConsoleManager(CommandManager commandManager){
+    public ConsoleManager(CommandManager commandManager) {
         this.commandManager = commandManager;
         this.scanner = new Scanner(System.in);
     }
 
-    public void interactiveMode() {
+    public void interactiveMode() throws Exception {
         System.out.println("Welcome to the City Manager!");
-        while(true) {
-            System.out.print("Enter a command: ");
-            String command = scanner.nextLine().trim();
-            commandManager.executeCommand(command);
+        try {
+            while (true) {
+                System.out.print("Enter a command: ");
+                String command = scanner.nextLine().trim();
+                if (command.isEmpty()) {
+                    System.out.println("Error: Command cannot be empty. Please try again.");
+                    continue;
+                }
+
+                if ("exit".equalsIgnoreCase(command)) {
+                    System.out.println("Exiting the program. Goodbye!");
+                    break;
+                }
```

Comment:
Why special case? there are better ways of handling exit without hardcoding it here